src/components/elements/Feedback.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import styled from 'styled-components'
+import { LIGHT_RED, FONT_FAMILY } from '../../config/styles'
+import Link from '../navigation/Link'
+import { QuestionMarkIcon } from '../icons'
+
+const FeedbackWrapper = styled.div`
```

Comment:
@shakersinthedark to be more semantic, what do you think about calling the "FeedbackWrapper" something like "StyledFeedback" because the main responsibility of this component is to provide the styles to the Feedback component. Wrapping the Feedback is a consequence of this main responsibility

Reply:
no worries, sounds good